CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - JSON API: use `\n\n` to terminate responses, for simplified parsing (pylightning now relies on this)
 - JSON API: `fundchannel` now includes an `announce` option, when false it will keep channel private. Defaults to true.
 - JSON API: `listpeers`'s `channels` now includes a `private` flag to indicate if channel is announced or not.
-- JSON API: `invoice` route hints may now include private channels if you have no public ones.
+- JSON API: `invoice` route hints may now include private channels if you have no public ones, unless new option `exposeprivatechannels` is false.
```

Comment:
Shall we call this `privatehints`? Still a mouthful, but maybe causes less confusion.

Reply:
But the hint isn't private.  It's awkward, an `exposeprivates` has unwanted connotations :)